.azure-pipelines.yaml

```diff
@@ -0,0 +1,69 @@
+pr: none
+
+jobs:
+- job:
+  displayName: Linux
+  pool:
+    vmImage: "ubuntu-16.04"
+
+  steps:
+  - bash: echo "##vso[task.prependpath]$CONDA/bin"
+    displayName: Add conda to PATH.
+
+  - bash: conda update conda --yes --quiet
+    displayName: Update conda.
+
+  - bash: conda env create
+    displayName: Create Anaconda environment.
+
+  - bash: |
+      source activate gettsim
```

Comment:
Why does that still work? With modern conda it should be `conda activate gettsim`, right?

Reply:
I think it is still supported, but in the future one should move to the other commands. The problem here is that ``conda init`` alters the configuration files which have to be resourced. That did not work when I tried it the last time.

Reply:
Cool, thanks. Looks to me from a very quick glance like that MS is well aware of this and `source activate` is their own workaround rather than old conda style. Can you add a comment to the yaml file pointing to: https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/anaconda